content/general/concepts/font-awesome/terms/fa-Nx/fa-Nx.md

```diff
@@ -0,0 +1,32 @@
+---
+Title: 'fa-Nx'
+Description: 'Where N can be a number from 2 to 10, adjusts the icon size relative to its container.'
```

Comment:
```suggestion
Description: 'Adjusts the icon size relative to its container.'
```

I don't think the parameter info is necessary in the `Description` for this entry.